src/main/java/athleticli/parser/ActivityParser.java

```diff
@@ -20,45 +19,87 @@
 public class ActivityParser {
     //@@author  AlWo223
     /**
-     * Parses the index of an activity.
+     * Parses the index of an activity from a string input.
      *
-     * @param commandArgs The raw user input containing the index.
-     * @return index                The parsed Integer index.
-     * @throws AthletiException If the input is not an integer.
+     * @param commandArgs       Raw user input containing the index.
+     * @return index            The parsed Integer index.
```

Comment:
```suggestion
     * @return                  The parsed Integer index.
```

`@return` field does not need to have a name like `@param`.